language/diem-vm/src/errors.rs

```diff
@@ -45,9 +45,6 @@ pub fn convert_prologue_error(
 ) -> Result<(), VMStatus> {
     let status = error.into_vm_status();
 
```

Comment:
@0o-de-lally 
I see the https://github.com/OLSF/libra/issues/760 but if we remove this line, we will not be able to see the real error now. I think we should remove this line after/if we solve https://github.com/OLSF/libra/issues/760.